letsencrypt/plugins/standalone.py

```diff
@@ -134,18 +134,12 @@ def supported_challenges_validator(data):
 
 
 class Authenticator(common.Plugin):
-    """Standalone Authenticator.
+    """Standalone Authenticator."""
 
-    This authenticator creates its own ephemeral TCP listener on the
-    necessary port in order to respond to incoming DVSNI and SimpleHTTP
-    challenges from the certificate authority. Therefore, it does not
-    rely on any existing server program.
-
-    """
     zope.interface.implements(interfaces.IAuthenticator)
     zope.interface.classProvides(interfaces.IPluginFactory)
 
-    description = "Standalone Authenticator"
+    description = "LE Will Automatically Test Your Domain"
```

Comment:
The problem with that is it's too long and cuts off the (standalone) text.
![lelong](https://cloud.githubusercontent.com/assets/649860/10737911/0882753e-7bd2-11e5-876b-3e282031656b.jpg)